client/src/main/scala/skuber/ext/HorizontalPodAutoscaler.scala

```diff
@@ -41,8 +41,8 @@ object HorizontalPodAutoscaler {
   }
    case class Spec( 
      scaleRef: SubresourceReference,
-     minReplicas: Int = 1,
-     maxReplicas: Int = 1,
```

Comment:
I believe maxReplicas is a required field, so propose leaving as-is if so (otherwise setting the value to None would probably result in an error when sent to Kubernetes).